unstructured/partition/auto.py

```diff
@@ -70,6 +71,9 @@ def partition(
     ocr_languages
         The languages to use for the Tesseract agent. To use a language, you'll first need
         to isntall the appropriate Tesseract language pack.
+    pdf_extract_tables
+        If True, in the case that the file to be processed is detected to be a PDF, any tables that
+        are detected will be extracted.
```

Comment:
this  kwarg name s is a bit confiusing imo, because a Table element gets created extracted in either case. (when true, the html string is also produced)